src/js/providers/html5.js

```diff
@@ -495,9 +495,16 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
                     currentQuality = i;
                 }
                 if (label && levels[i].label === label) {
-                    return i;
+                    currentQuality = i;
+                    break;
                 }
             }
+            if (!currentQuality) {
+                // Choose the highest quality
+                const sorted = levels.map((level, index) => Object.assign({ index, height: 0 }, level))
+                    .sort((a, b) => b.height - a.height);
+                currentQuality = sorted[0].index;
+            }
```

Comment:
I removed this from the Tizen branch merged into master.  It was a hack to pick the best mp4 quality when HLS is not selected. We shouldn't add code like this to the player and should fix HLS supports.

Reply:
This looks like a bad rebase on my part. Will fix.